### PR TITLE
Cv 85 add new properties to legal entity messages

### PR DESCRIPTION
**Pull request by PeterDavidWilliams**

### CODE REVIEW

File: src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CreateAccountCommandTests/WhenICallCreateAccount.cs [hunk 1]
```diff
@@ -297,7 +299,13 @@ public async Task ThenAAddedLegalEntityEventIsPublished()
             const string organisationName = "Org";
 
             //Arrange
-            var createAccountCommand = new CreateAccountCommand { PayeReference = "123EDC", AccessToken = "123rd", RefreshToken = "45YT", OrganisationStatus = "active", OrganisationName = organisationName, ExternalUserId = _user.Ref.ToString() };
```
Comment (Pauljgraham) [inline src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CreateAccountCommandTests/WhenICallCreateAccount.cs]: Could we use TestCase("Charity", "Charity from Messages Type")
... and the other mappings. Just covers several scenarios

Comment (PeterDavidWilliams) [reply src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CreateAccountCommandTests/WhenICallCreateAccount.cs]: Resolved by https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/pull/1556/commits/dc01dbe0a97a0c0730c3134f1ae394fffb748e31

File: src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CreateLegalEntityCommandTests/WhenICallCreateLegalEntity.cs [hunk 2]
```diff
@@ -228,7 +231,10 @@ public async Task ThenAddedLegalEntityEventIsPublished()
                 e.AccountLegalEntityPublicHashedId.Equals(ExpectedAccountLegalEntityPublicHashString) &&
                 e.OrganisationName.Equals(_command.Name) &&
                 e.UserName.Equals(_owner.FullName()) &&
-                e.UserRef.Equals(Guid.Parse(_owner.UserRef)))));
+                e.UserRef.Equals(Guid.Parse(_owner.UserRef)) &&
+                e.OrganisationReferenceNumber.Equals(_agreementView.LegalEntityCode) &&
+                e.OrganisationAddress.Equals(_agreementView.LegalEntityAddress) &&
+                e.OrganisationType.ToString().Equals(_agreementView.LegalEntitySource.ToString())))); 
             //e.Created.)));
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CreateLegalEntityCommandTests/WhenICallCreateLegalEntity.cs]: Can we remove this commented line?

Comment (PeterDavidWilliams) [reply src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CreateLegalEntityCommandTests/WhenICallCreateLegalEntity.cs]: Addressed by https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/pull/1556/commits/d4498ca99b190f639df1175d410c63e7aeb104ba

**Pull request closed without merging**